File: 1-Iniciante/1037-Intervalo-3/1037.py

onun=float(input())

if onun>=0 and onun <=25:
    print("Intervalo [0,25]")

elif onun>25 and onun <=50:
    print("Intervalo (25,50]")

elif onun>50 and onun <=75:
    print("Intervalo (50,75]")

elif onun>75 and onun <=100:
    print("Intervalo (75,100]")

else:
    print("Fora de intervalo")

File: 1-Iniciante/1037-Intervalo-3/1037.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        double onun = double.Parse(Console.ReadLine());
        
        if (onun>=0 & onun <=25){
            Console.WriteLine("Intervalo [0,25]");
        }
        else if (onun>25 & onun <=50){
            Console.WriteLine("Intervalo (25,50]");
        }
        else if (onun>50 & onun <=75){
            Console.WriteLine("Intervalo (50,75]");
        }
        else if (onun>75 & onun <=100){
            Console.WriteLine("Intervalo (75,100]");
        }
        else{
            Console.WriteLine("Fora de intervalo");
        }
    }
}